### PR TITLE
refactor: for investigate problem on stackblitz

### PR DESCRIPTION
**Pull request by splincode**
## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows [Conventional Commits](https://www.conventionalcommits.org/en/)
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [ ] Bugfix
- [ ] Feature
- [x] Refactoring
- [ ] Code style update
- [ ] Build or CI related changes
- [ ] Documentation content changes

### CODE REVIEW

File: projects/cdk/directives/element/downlevel-element.ts [hunk 1]
```diff
@@ -0,0 +1,7 @@
+import {ElementRef} from '@angular/core';
+
+export function tuiDownlevelElement<T extends Element = HTMLElement>(
+    elementRef: ElementRef<T>,
```
Comment (waterplea) [inline projects/cdk/directives/element/downlevel-element.ts:4]: Destruct, please

File: projects/cdk/directives/element/element.directive.ts [hunk 2]
```diff
@@ -1,11 +1,17 @@
...
-export class TuiElementDirective<T extends Element = HTMLElement> extends ElementRef<T> {
-    constructor(@Inject(ElementRef) {nativeElement}: ElementRef<T>) {
-        super(nativeElement);
+export class TuiElementDirective<T extends Element = HTMLElement>
+    implements ElementRef<T>
+{
+    nativeElement!: T;
+
+    constructor(@Inject(ElementRef) elementRef: ElementRef<T>) {
+        return tuiDownlevelElement(elementRef);
```
Comment (waterplea) [inline projects/cdk/directives/element/element.directive.ts:15]: How about we inline this return here instead, without extra function? And also add comment why we have to do this.

File: projects/cdk/directives/element/downlevel-element.ts (hunk 1 above)
Comment (splincode) [reply projects/cdk/directives/element/downlevel-element.ts]: fixed

File: projects/cdk/directives/element/element.directive.ts (hunk 2 above)
Comment (splincode) [reply projects/cdk/directives/element/element.directive.ts]: fixed

**Review by waterplea — APPROVED** [review]

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**